### PR TITLE
Feature/team-members-database-table

### PR DESCRIPTION
**Pull request by RPinkha**
## Describe your changes

Create a new model/team-member.ts file to create a database in keystone for team members when running the scripts npm run generate && npm run dev
Add export * from './team-member.ts'; inside of index.ts file

Team Members

image

## Issue ticket number and link

EES-26
https://tripleten-apiary.atlassian.net/browse/EES-26

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If there are changes to custom resolvers, I have added / updated automated tests
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
+input TeamMemberWhereUniqueInput {
   id: ID
-  email: String
 }
 
-input UserWhereInput {
-  AND: [UserWhereInput!]
-  OR: [UserWhereInput!]
-  NOT: [UserWhereInput!]
+input TeamMemberWhereInput {
```
Comment (michaelabudde) [inline schema.graphql]: The naming of the input and values all make sense and seem intuitive, nice!

**Review by michaelabudde — APPROVED** [review]
Overall, this code looks good, the access control rules look correct, and it looks to store all necessary team member information. I believe this is good to go, nice work!

**Comment by RPinkha** [comment]
@tomershvueli pulled from main and resolved conflicts.

**Review by tomershvueli — APPROVED** [review]
Looks great and I'm able to upload images to S3!

**Pull request merged**